### PR TITLE
Patch/fix build 10.0.2 46658 error

### PR DESCRIPTION
**Pull request by kalvin807**
@Cidan
Hi, thank you for the great mod and all the maintenance!

This PR fixes errors caused by API changes since 10.0.2. I tested it locally and it seems to have worked well so far. 
Please review it and let me know any things you want to change. Really excited to release this asap as this mod is really essential to me :P

@MleveK Thank you for your patches!! I do look at your source and take several references. will be great if you can take a look at this!

Stuff I tested (manually)
- [x] Bags
- [x] Settings

### CODE REVIEW

File: widgets/ItemButton.lua [hunk 1]
```diff
@@ -639,7 +655,8 @@ end
 function stackProto:UpdateCount()
 	local count = 0
 	for slotId in pairs(self.slots) do
-		count = count + (select(2, GetContainerItemInfo(GetBagSlotFromId(slotId))) or 1)
+		itemCount = SafeGetItem(GetContainerItemInfo(GetBagSlotFromId(slotId)), "stackCount") or 1
+s
```
Comment (set4812) [inline widgets/ItemButton.lua:659]: `s` mistake?

Comment (kalvin807) [reply widgets/ItemButton.lua]: Nice catch! I removed the whole line ... now it is fixed

**Comment by jnwhiteh** [comment]
Hey @kalvin807 this is going to break things on Classic and Wrath pretty heavily. Take a look at #788 and see if there's anything I missed there. That's properly guarded for the various versions.

**Comment by kalvin807** [comment]
> Hey @kalvin807 this is going to break things on Classic and Wrath pretty heavily. Take a look at #788 and see if there's anything I missed there. That's properly guarded for the various versions.

Thank you for your MR. Yours is definitely more robust and tested in various versions. I will be closing this MR.

**Pull request closed without merging**